supplements/critical-role-call-of-the-netherdeep/items.xml

```diff
@@ -0,0 +1,624 @@
+			<p class="indent"><strong><em>Exalted State.</em></strong> In this state, the jewel has received the blessing of Corellon the Arch Heart. A gleaming emerald surrounded by a halo of gold appears on the jewel.</p>
+			<p class="indent">The following benefits of the jewel improve:</p>
+			<ul>
+				<li>The bonus that the jewel confers to your AC increases to +3.</li>
+				<li>Its number of charges increases to 7.</li>
+			</ul>
+			<p class="indent">The jewel gains the following additional properties, which you can use while wearing or holding it:</p>
+			<ul>
+				<li>You gain the ability to breathe water, and you gain a swimming speed equal to your walking speed.</li>
+				<li>Each of your allies within 30 feet of you gains the ability to breathe water and gains a swimming speed equal to its walking speed.</li>
+				<li>As a bonus action, you can expend 1 of the jewel's charges to target yourself or one willing creature you can see within 15 feet of yourself. The target teleports to an unoccupied space of your choice within 15 feet of yourself, along with any equipment the target is wearing or carrying. The target appears in a flash of golden radiance, and each creature of your choice within 5 feet of the target's new location must make a DC 18 Constitution saving throw. On a failed save, the creature takes 4d10 radiant damage and is blinded until the start of your next turn. On a successful save, the creature takes half as much damage and isn't blinded.</li>
+			</ul>
+			<div class="sidebar">
+				<h5>VESTIGES OF DIVERGENCE</h5>
+				<p>When the great wars of the Calamity rolled across Exandria, heroes of divinity accepted legendary relics from their patrons and gods. At the same time, archmages wove dangerous arcane power into items of immense magical might. When the war ended, most of these magic items were buried with their wielders beneath ash and dust. Those items that remained accessible were passed down as heirlooms. These remnants of a lost era are known as the Vestiges of Divergence</p>
+				<p class="indent">Each Vestige of Divergence is a magic item that grows in power over time. When it is first encountered, the item is usually in its Dormant State. Either on its own or in response to the heroic deeds of its bearer, a Vestige of Divergence can enter its Awakened State. In time, it can reach its ultimate potential and enter its Exalted State, usually in response to a momentous event or some heroic achievement on the part of its bearer. The properties gained at each new state are cumulative with the properties of a previous state, unless otherwise noted.</p>
+			</div>
+		</description>
+		<setters>
+			<set name="category">Wondrous Items</set>
+			<set name="type">Wondrous Item</set>
+			<set name="rarity">Legendary</set>
+			<set name="keywords">vestige of divergence, ac, invisibility, reaction, bonus action</set>
+			<set name="attunement">true</set>
+			<set name="charges">7</set>
+			<set name="cost" currency="gp">0</set>
```

Comment:
```suggestion
			<set name="cost" currency="gp">0</set>
			<set name="slot">misc</set>
```
There is a bug with items that require attunement and are not equip-able that sometimes the effect is kept even without the items.  The app also lags more without it